pwiz_tools/Skyline/Model/Databinding/Entities/PeptideResult.cs

```diff
@@ -267,6 +267,15 @@ public FiguresOfMerit BatchFiguresOfMerit
             }
         }
 
+        public int PeptideSpectrumMatchCount
+        {
+            get
+            {
+                return SrmDocument.Settings.GetRetentionTimes(GetResultFile().ChromFileInfo.FilePath,

```

Comment:
I can imagine wanting the total number of matches for a peptide regardless of matching the result file, can't you? This is what I thought you had implemented until I read the code. Now, I definitely think both are interesting and convey different useful information. It would definitely make naming trickier, because you would want this name for the total and something even longer for this like "Peptide Spectrum Match In File Count" or "Peptide Spectrum Match Count In File"